packages/common/src/groups/HubGroups.ts

```diff
@@ -208,12 +210,14 @@ export async function updateHubGroup(
   await updateGroup(opts);
 
   // create or update entity settings
-  const entitySetting = await createOrUpdateEntitySettings(
-    hubGroup,
-    context.hubRequestOptions
-  );
-  hubGroup.entitySettingsId = entitySetting.id;
-  hubGroup.discussionSettings = entitySetting.settings.discussions;
+  if (!context.isPortal) {
+    const entitySetting = await createOrUpdateEntitySettings(
+      hubGroup,
+      context.hubRequestOptions
+    );
+    hubGroup.entitySettingsId = entitySetting.id;
+    hubGroup.discussionSettings = entitySetting.settings.discussions;
+  }
```

Comment:
We likely also need to prevent any fetches from occurring in enterprise sites. The above changes only prevent creates/updates